### PR TITLE
Support installation via pipx

### PR DESCRIPTION
**Pull request by jstvz**
This pull request adds simplified installation instructions for Windows 10
using `pipx`. This utility automates the creation of a virtualenv, the
installation of `cumulusci`'s dependencies, and linking the `cci.exe`
executable in a known location on the user's PATH.

# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

**Comment by jstvz** [comment]
Supersedes #995

File: docs/tutorial.rst [hunk 1]
```diff
@@ -14,6 +14,17 @@ On macOS and Linux, the easiest way to install CumulusCI is using `homebrew <htt
     $ brew tap SFDO-Tooling/homebrew-sfdo
     $ brew install cumulusci
 
+On Windows 10, the easiest way to install CumulusCI is using ```pipx`` <https://github.com/pipxproject/pipx>`__: 
+
+1. In a new command prompt, run: ``python -m pip install --user pipx``
```
Comment (davisagli) [inline docs/tutorial.rst]: This is assuming that the user already has the correct version of Python installed, but we don't tell them how to do that until further down.

File: docs/tutorial.rst [hunk 2]
```diff
@@ -14,6 +14,17 @@ On macOS and Linux, the easiest way to install CumulusCI is using `homebrew <htt
...
+2. Add the following paths to your ``PATH`` environment variable:
```
Comment (davisagli) [inline docs/tutorial.rst]: We probably need to give instructions on how to do that.

File: docs/tutorial.rst [hunk 3]
```diff
@@ -14,6 +14,17 @@ On macOS and Linux, the easiest way to install CumulusCI is using `homebrew <htt
...
+
+   1. ``%USERPROFILE%\AppData\Roaming\Python\Python37\Scripts``
+   2. ``%USERPROFILE%\.local\bin``
+
+3. In a new command prompt, run: ``pipx install cumulusci``
+4. In a new command prompt, verify by running: ``cci version``
```
Comment (davisagli) [inline docs/tutorial.rst]: We should say what they should expect to see.

File: docs/tutorial.rst [hunk 4]
```diff
@@ -33,56 +24,70 @@ Install Python 3: https://www.python.org/downloads/windows/
...
-    $ python3 -m venv cci
+Install CumulusCI
+-----------------
 
-This creates a virtualenv in a new folder named ``cci``. Whenever you want to work with commands installed in this virtualenv, you must first activate it:
+macOS/Linux
+^^^^^^^^^^^
 
-.. code-block:: console
+On macOS and Linux, the easiest way to install CumulusCI is using `homebrew <https://docs.brew.sh/>`_  and `linuxbrew <https://linuxbrew.sh/>`_, respectively:
```
Comment (davisagli) [inline docs/tutorial.rst]: This will use a homebrew-installed Python, right? Should we remove the "Install Python 3" link for macOS above?

Comment (jstvz) [reply docs/tutorial.rst]: Ah, good catch. I'll remove that macOS/Linux from that section, then move it under Windows.

Also, noticed that they merged Linuxbrew back in 2.0.0, so I'll remove any of those references.

**Pull request merged**